src/local/butler/py_unittest.py

```diff
@@ -128,20 +129,26 @@ def test_worker_init():
 
 def run_one_test_parallel(args):
   """Test worker."""
-  os.environ['PARALLEL_TESTS'] = '1'
-
-  test_modules, suppress_output = args
-  suite = unittest.loader.TestLoader().loadTestsFromNames(test_modules)
-
-  stream = io.BytesIO()
-
-  # Verbosity=0 since we cannot see real-time test execution order when tests
-  # are executed in parallel.
-  result = unittest.TextTestRunner(
-      stream=stream, verbosity=0, buffer=suppress_output).run(suite)
-
-  return TestResult(stream.getvalue(), len(result.errors), len(result.failures),
-                    len(result.skipped), result.testsRun)
+  try:
+    os.environ['PARALLEL_TESTS'] = '1'
+
+    test_modules, suppress_output = args
+    suite = unittest.loader.TestLoader().loadTestsFromNames(test_modules)
+
+    stream = io.BytesIO()
+
+    # Verbosity=0 since we cannot see real-time test execution order when tests
+    # are executed in parallel.
+    result = unittest.TextTestRunner(
+        stream=stream, verbosity=0, buffer=suppress_output).run(suite)
+
+    return TestResult(stream.getvalue(),
+                      len(result.errors), len(result.failures),
+                      len(result.skipped), result.testsRun)
+  except:
```

Comment:
except Exception:

Reply:
This was intentional, since not all exceptions are necessarily derived from Exception.

Reply:
actually, BaseException probably covers all that we care about here (including system exiting ones). will use that